### DISCUSSION TITLE
Is there anyway to show all the future time?

### DISCUSSION THREAD

**Discussion by huypham1411** [tradingview/lightweight-charts#1590] [Q&A]
Hi,
Is there any way to show all the future time value or I need to extend the chart data with the time value?
![image](https://github.com/tradingview/lightweight-charts/assets/54835897/3e53f31c-f8dd-4bb1-a4be-0cdce07b41d9)
I tried this 
![image](https://github.com/tradingview/lightweight-charts/assets/54835897/08ae73a5-c75d-45c4-b13a-d25c1549a39a)
but 
what I want to do is able to move the drawn object outside the chart zone
https://streamable.com/kvegn5
here is lightweight demo https://tradingview.github.io/lightweight-charts/plugin-examples/plugins/rectangle-drawing-tool/example/
I want to draw rectangle outside the time range, is there any way to do it?
Currently, I cannot move the drawn object outside the chart data zone as the video above.
I would appreciate any help!

**Comment by manhdung3010** [comment]
I'm also having this problem, do you have a solution?

**Reply by huypham1411** [reply]
still don't have it, I tried to generate more data to extend the time axis but the drawn object cannot be moved outside the data range, only able if all the other series have full data.
